flink-yarn/src/main/java/org/apache/flink/yarn/configuration/YarnConfigOptions.java

```diff
@@ -79,14 +87,18 @@
 	 */
 	public static final ConfigOption<String> APPLICATION_ATTEMPTS =
 		key("yarn.application-attempts")
-		.noDefaultValue();
+		.noDefaultValue()
```

Comment:
We don't really recommend to leave this at `1` anymore, do we?

Reply:
I haven't seen any recommendation in the [Yarn docs](https://ci.apache.org/projects/flink/flink-docs-master/ops/deployment/yarn_setup.html). In the [HA docs](https://ci.apache.org/projects/flink/flink-docs-master/ops/jobmanager_high_availability.html#yarn-cluster-high-availability) we set it to 10 but never explicitly call this a recommended value.